### ISSUE TITLE
Vanity generator shows wrong address

### ISSUE THREAD

**Issue by olehmisar** [dethcrypto/dethtools#96]
Steps to reproduce:
1. Go to https://tools.deth.net/vanity-address-generator
2. Click generate
3. Import generated private key into metamask
4. Address in metamask is different from the address shown on the website

**Comment by 7sne** [comment]
Yeah, apparently it doesn't work. I will figure it out, thanks for the feedback.
